srtcore/md5.h

```diff
@@ -50,6 +50,8 @@
 #ifndef md5_INCLUDED
```

Comment:
It would be nice that you place a comment with the information that all symbols have been put under the srt namespace to avoid potential linkage conflicts. This wouldn't be normally necessary, but here is the code that have been taken from a 3rd party and it was intended to be brought in unmodified - hence it would be reasonable to simply mention that this namespace is the only modification introduced here.

Reply:
Done.

Reply:
Very good, thans.